### PR TITLE
Avivash/use npm

### PR DESCRIPTION
**Pull request by icidasset**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
-
 ## Usage
 
 Uses Ethereum by default with `window.ethereum` as the provider. Currently only works with MetaMask because it's the only wallet with encryption and decryption.
 
 ```ts
-import * as walletauth from "webnative-walletauth"
-import { AppScenario } from "webnative"
+import * as walletauth from "webnative-walletauth";
+import { Scenario } from "webnative";
```
Comment (icidasset) [inline README.md:9]: @avivash `AppScenario` was correct as before, `Scenario` is a slightly different object used for permissioned apps (see https://webnative.fission.app/modules.html#permissionedApp)

Comment (avivash) [reply README.md:9]: > @avivash `AppScenario` was correct as before, `Scenario` is a slightly different object used for permissioned apps (see https://webnative.fission.app/modules.html#permissionedApp)

strange 🤔, it didn't seem to work when I tried with `AppScenario`. Is that possibly in an unreleased version? 
image

Comment (icidasset) [reply README.md:9]: Hmm weird, it's in 0.34.x https://webnative.fission.app/enums/AppScenario.html
Does it keep saying that after reloading your TS language server?

Comment (avivash) [reply README.md:9]: > Hmm weird, it's in 0.34.x https://webnative.fission.app/enums/AppScenario.html Does it keep saying that after reloading your TS language server?

Yep, still doesn't work after a TS server restart, unfortunately

Comment (avivash) [reply README.md:9]: I'll push the readme update separately 👌🏼

Comment (avivash) [reply README.md:9]: Oh nevermind, I see you already did! 🙌🏼

Comment (icidasset) [reply README.md:9]: I already changed it back, all good 👍

**Pull request merged**